network-api/networkapi/wagtailpages/tests/research_hub/test_landing_page.py

```diff
@@ -32,3 +39,37 @@ def test_get_latest_research_pages_does_not_return_private_page(self):
         self.assertEqual(len(latest_research_pages), 1)
         self.assertIn(detail_page_public, latest_research_pages)
         self.assertNotIn(detail_page_private, latest_research_pages)
+
+    def test_get_latest_research_pages_returns_detail_pages_of_same_locale(self):
+        """
+        Ensure that the latest research pages are returned in the same locale as the landing page.
+        """
+        # Create a detail page in the default locale
+        self.create_research_detail_page()
+        # Sync the tree to ensure that the detail page is created in the fr locale
+        self.synchronize_tree()
```

Comment:
TIL: didn't know about this, good to know :)

Reply:
It's emulating the `$ python network-api/manage.py sync_locale_trees` cron job that runs on production every hour to make sure pages get synced to all locales.